src/library/basetracktablemodel.cpp

```diff
@@ -159,6 +123,21 @@ void BaseTrackTableModel::initTableColumnsAndHeaderProperties(
     m_columnCache.setColumns(tableColumns);
     if (m_columnHeaders.size() < tableColumns.size()) {
```

Comment:
We want that the m_columnHeaders is always current. So I think this condition needs to be removed.

Reply:
I didn't notice that re-initialized `m_columnCache` is accessed in the loop.